index.html

```diff
@@ -0,0 +1,371 @@
+                </ul>
+                <div class="menu-btn">
+                    <i class="fas fa-bars"></i>
+                </div>
+        </div>
+    </nav>
+
+
+
+
+    <!-------------home------------------>
+    <secttion class="home" id="home">
+        <div class="container">
+            <div class="left">
+                <div class="home-text">
+
+                    <div class="text1"><h2>hello i'm</h2></div>
+                    <div class="text2"><h1>my name</h1></div>
+                    <div class="text3"><h3>working as this</h3></div>
+                    
+                </div>
+                <div class="social">
+
+                    <div class="icon">
+                        <a href="#"><i class="fab fa-twitter-square"></i></a>
+                    </div>
+                    <div class="icon">
+                        <a href="#"><i class="fab fa-linkedin"></i></a>
+                    </div>
+                    <div class="icon">
+                        <a href="#"><i class="fab fa-facebook-square"></i></a>
+                    </div>
+
+
+                    
+                    
+                    
+                </div>  
+            </div>
+            <div class="right">
+                <div class="image-box">
+                    <img src="images/home.png" alt="">
+                </div>
+            </div>
+        </div>
+    </secttion>
+<!---------------end of home---------------->
+<!----------------about mr--------------------->
+    <section class="about" id="about">
+        
+        <div class="header">
+            <h1>about me</h1>
+        </div>
+        <div class="container">   
+            
+            <div class="left">
+                <div class="image-box">
+                    <img src="images/home.png" alt="">
+                </div>
+                <div class="about-text">
+                    <p>Lorem ipsum dolor sit amet, consectetur
+                         adipisicing elit. Obcaecati, in ut? 
+                         Tenetur itaque cum voluptatibus in et
+                          provident recusandae maiores corpori
+                          s culpa, quae, inventore odit, velit
+                           quidem facilis expedita sunt?
+                    </p>
+                </div>   
+            </div>
+            <div class="right">
+                <h2>My Skills</h2>
+                <div class="skills">
+                    <div class="skill-box">html</div>
+                    <div class="skill-box">css</div>
+                    <div class="skill-box">javascript</div>
+                    <div class="skill-box">php</div>
+                    <div class="skill-box">phython</div>
+                    <div class="skill-box">mysql</div>
+                    <div class="skill-box">c++</div>
+                    <div class="skill-box">photoshop</div>
+                </div>
+                <h2>language proficiency</h2>
+                <div class="language">

```

Comment:
shall we remove the language proficiency from here? alter for my skills we can separate it as core skills and additional skills. Under core skills, we can add the specialized like machine learner, java developer etc, under additional skills we can add other skills like HTML, PHP etc.